databases/utils.py

```diff
@@ -20,7 +20,7 @@
     'array_push',
     'json_arrays',
     'raw_queries',
-    'create_many',
+    'skip_duplicates',
```

Comment:
nit: I think this should be `create_many_skip_duplicates` and would be great to rename the test files as well :)